lib/rules/length-zero-no-unit/index.js

```diff
@@ -65,6 +65,10 @@ const rule = function(actual, secondary, context) {
 		});
 
 		root.walkAtRules((atRule) => {
+			if (atRule.variable && !hasBlock(atRule)) {
```

Comment:
Could you add comment before this condition to specify that this condition is for Less variables.

Reply:
Done!